src/main/java/Runner.java

```diff
@@ -25,17 +27,21 @@
             new MilitaryPlane("C-130 Hercules", 650, 5000, 110000, MilitaryType.TRANSPORT)
     );
 
-    public static void main(String[] args) throws NonMilitaryPlainException {
-        Airport airport = new Airport(planes);
-        Airport militaryAirport = new Airport(airport.getMilitaryPlanes());
-        Airport passengerAirport = new Airport(airport.getPassengersPlane());
-        System.out.println("Military airport sorted by max distance: " + militaryAirport
-                .sortByMaxDistance()
-                .toString());
-        System.out.println("Passenger airport sorted by max speed: " + passengerAirport
-                .sortByMaxSpeed()
-                .toString());
-
-        System.out.println("Plane with max passenger capacity: " + passengerAirport.getPassengerPlaneWithMaxPassengersCapacity());
+    public static void main(String[] args) {
+        try {
+            Airport airport = new Airport(planes);
+            Airport militaryAirport = new Airport(airport.getMilitaryPlanes());
+            Airport passengerAirport = new Airport(airport.getPassengersPlane());
+            LOGGER.info("Military airport sorted by max distance: " + militaryAirport
+                    .sortByMaxDistance(planes)
+                    .toString());
+            LOGGER.info("Passenger airport sorted by max speed: " + passengerAirport
+                    .sortByMaxSpeed(planes)
+                    .toString());
+            LOGGER.info("Plane with max passenger capacity: " + passengerAirport.getPassengerPlaneWithMaxPassengersCapacity());
+        } catch (NonMilitaryPlainException e) {
+            LOGGER.info(e.getMessage());
```

Comment:
info logger level is for info messages. here you're treating an exception. so it's LOGGER.error